src/main/java/io/reactivex/observers/BaseTestConsumer.java

```diff
@@ -567,23 +567,23 @@ public final U assertValueSet(Collection<? extends T> expected) {
     @SuppressWarnings("unchecked")
     public final U assertValueSequence(Iterable<? extends T> sequence) {
         int i = 0;
-        Iterator<T> vit = values.iterator();
-        Iterator<? extends T> it = sequence.iterator();
+        Iterator<T> actualIterator = values.iterator();
+        Iterator<? extends T> expectedIterator = sequence.iterator();
         boolean actualNext;
         boolean expectedNext;
         for (;;) {
-            actualNext = it.hasNext();
-            expectedNext = vit.hasNext();
+            expectedNext = expectedIterator.hasNext();
+            actualNext = actualIterator.hasNext();
 
             if (!actualNext || !expectedNext) {
                 break;
             }
 
-            T v = it.next();
-            T u = vit.next();
+            T v = expectedIterator.next();
+            T u = actualIterator.next();
 
             if (!ObjectHelper.equals(u, v)) {
-                throw fail("Values at position " + i + " differ; Expected: " + valueAndClass(u) + ", Actual: " + valueAndClass(v));
+                throw fail("Values at position " + i + " differ; Expected: " + valueAndClass(v) + ", Actual: " + valueAndClass(u));
```

Comment:
This change also needs a test

Reply:
There is no behavior change here, just a swapped variable name. I'll restore the original u - expected, v - actual naming

Reply:
Ah. Well at least we have a valid reason to start working on RxJava 3 — kill these one-letter variable names :trollface: